authors/apps/authentication/serializers.py

```diff
@@ -140,3 +140,57 @@ def update(self, instance, validated_data):
         instance.save()
 
         return instance
+
+
+class PasswordResetSerializer(serializers.ModelSerializer):
+    """
+    Serializer for requesting a password reset e-mail.
+    """
+    email = serializers.EmailField()
+
+    @classmethod
+    def validate_email(self, value):
+        try:
+            valid_email = User.objects.get(email=value)
+        except Exception:
+            raise serializers.ValidationError(f"{value} email is Invalid")
+        return value
+
+    class Meta:
+        model = User
+        fields = ['email']
+
+
+class ConfirmPassword(serializers.ModelSerializer):
```

Comment:
@ManuelDominic, there's no much need to verify the password in the backend api.
This can be done in the frontend

Reply:
@kimbugp, i have used that to ensure a user provides both password one and two as the format to verify the user remembers what he/she has provided `(password match)` then saved.
I have renamed the class to bring out some meaning.

Reply:
Checking if two password fields match during a sign up should be purely done with client-side logic. It is provided as a safety against a user mistakenly inserting a typo into their password. A server-side check is pointless, as your client will have prevented it and if your user is a tech savvy individual that does everything with curl then it's on them if they mess up.